### PR TITLE
Dev ops

### PR DESCRIPTION
**Pull request by tomerXarchimedes**
Built the basic of local machine devops pipeline
* everything runs on a local fork of mainnet
* integration test skeleton

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -19,12 +19,12 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
...
-//      forking: {
-//        url: "https://eth-mainnet.alchemyapi.io/v2/<key>",
-//        blockNumber: 11095000
-//      }
-//    }
-//  }
+  networks: {
+    hardhat: {
+      forking: {
+        url: "https://eth-mainnet.alchemyapi.io/v2/NfWNo_4fJbF5Aiu_KD-j4qVeyn1X_xqs",
```
Comment (yotamdaniel) [inline hardhat.config.js:25]: Need to remove key

File: test/0testSuitCheck.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
...":"address"}],"name":"Deprecate","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"feeBasisPoints","type":"uint256"},{"indexed":false,"name":"maxFee","type":"uint256"}],"name":"Params","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"_blackListedUser","type":"address"},{"indexed":false,"name":"_balance","type":"uint256"}],"name":"DestroyedBlackFunds","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"_user","type":"address"}],"name":"AddedBlackList","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"_user","type":"address"}],"name":"RemovedBlackList","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"owner","type":"address"},{"indexed":true,"name":"spender","type":"address"},{"indexed":false,"name":"value","type":"uint256"}],"name":"Approval","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"from","type":"address"},{"indexed":true,"name":"to","type":"address"},{"indexed":false,"name":"value","type":"uint256"}],"name":"Transfer","type":"event"},{"anonymous":false,"inputs":[],"name":"Pause","type":"event"},{"anonymous":false,"inputs":[],"name":"Unpause","type":"event"}]
```
Comment (yotamdaniel) [inline test/0testSuitCheck.js:28]: We'll need these ABI for more tests, maybe move them out to their own file

File: test/0testSuitCheck.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    const usdtToken = new ethers.Contract(USDT_ADDRESS, USDT_ABI, signer)  
+    expect(usdtToken.address).to.equal(USDT_ADDRESS);
+
+    // check decimals (USDT has 6 decimals)
+    let decimals = await usdtToken.decimals()
+    expect(decimals).to.equal(6);
+    })
+  });
+
+// USDT mainnet address + ABI
```
Comment (jasonstockman) [inline test/0testSuitCheck.js:26]: Should we move these variables to the top near "signer"?

File: test/MainnerHelper.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    /* helper functions */
+    helperResetNetwork: async function (lockBlock) {
+
+        // Reset hardhat mainnet fork	
+	    await network.provider.request({
+	    	method: "hardhat_reset",
+		    params: [
+    			{
+				    forking: {
+					    jsonRpcUrl: "https://eth-mainnet.alchemyapi.io/v2/NfWNo_4fJbF5Aiu_KD-j4qVeyn1X_xqs",
```
Comment (yotamdaniel) [inline test/MainnerHelper.js:15]: Same with key

File: test/MainnerHelper.js [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
...":"view","type":"function"},{"inputs":[],"name":"symbol","outputs":[{"internalType":"string","name":"","type":"string"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"totalSupply","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"address","name":"_to","type":"address"},{"internalType":"uint256","name":"_value","type":"uint256"}],"name":"transfer","outputs":[{"internalType":"bool","name":"","type":"bool"}],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"_from","type":"address"},{"internalType":"address","name":"_to","type":"address"},{"internalType":"uint256","name":"_value","type":"uint256"}],"name":"transferFrom","outputs":[{"internalType":"bool","name":"","type":"bool"}],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"_newGovernor","type":"address"}],"name":"transferGovernance","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[],"name":"vaultAddress","outputs":[{"internalType":"address","name":"","type":"address"}],"stateMutability":"view","type":"function"}]
```
Comment (yotamdaniel) [inline test/MainnerHelper.js:115]: Move ABI to its own file

**Review by yotamdaniel — CHANGES_REQUESTED** [review]
- Minor comments about where ABI should be 
- move key

File: hardhat.config.js (hunk 1 above)
Comment (tomerXarchimedes) [reply hardhat.config.js:25]: Done - thanks!

File: test/MainnerHelper.js (hunk 4 above)
Comment (tomerXarchimedes) [reply test/MainnerHelper.js:15]: fixed. thanks

File: test/0testSuitCheck.js (hunk 3 above)
Comment (tomerXarchimedes) [reply test/0testSuitCheck.js:26]: Yep. Moved it to a different file

File: test/0testSuitCheck.js (hunk 2 above)
Comment (tomerXarchimedes) [reply test/0testSuitCheck.js:28]: Good point. Done

File: test/MainnerHelper.js (hunk 5 above)
Comment (tomerXarchimedes) [reply test/MainnerHelper.js:115]: yep

**Review by yotamdaniel — COMMENTED** [review]
looks good to me

**Review by yotamdaniel — APPROVED** [review]

**Pull request merged**